packages/cashier/src/components/cashier-onboarding/__tests__/cashier-onboarding-side-note.spec.tsx

```diff
@@ -2,6 +2,14 @@ import React from 'react';
 import { fireEvent, render, screen } from '@testing-library/react';
 import CashierOnboardingSideNote from '../cashier-onboarding-side-note';
 
+declare global {
+    interface Window {
+        LC_API: {
+            open_chat_window: VoidFunction;
+        };
+    }
+}
```

Comment:
```suggestion
import '../../../../index';
```
@vinu-deriv You can import the declaration file instead of redefining it here, But there should be a better way to expose globals to jest environment, I will investigate and will share if I found anything 🙇🏻